### ISSUE TITLE
Analytical Table: CTA buttons/checkbox of a cell requires two tabs to receieve focus.

### ISSUE THREAD

**Issue by samanr** [SAP/ui5-webcomponents-react#5674]
### Describe the bug

The contents of a cell within a analytical table receive focus after two tabs. The first time the focus is on the cell and upon further tab it reaches the checkbox or a button.

### Isolated Example

https://stackblitz.com/edit/github-mseb8i?file=src%2FApp.tsx

### Reproduction steps

1. Add an analytical table with one of the cell containing a button or a checkbox
2. Apply focus on the cell adjacent to it and press tab
3. The focus goes to the cell first instead of the checkbox/button
4. Press tab again
5. Focus is on the button/checkbox

### Expected Behaviour

Focus should go directly to the interactive element inside the cell. 

### Screenshots or Videos

![Screenshot 2024-04-05 at 2 03 52 PM](https://github.com/SAP/ui5-webcomponents-react/assets/22089053/a1330f66-092e-48c2-910d-af3a873e4489)

### UI5 Web Components for React Version

1.26.0

### UI5 Web Components Version

1.23.0

### Browser

Chrome

### Operating System

Mac Os Sonoma 14.3.1

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @samanr 

This behavior is intended. As developers can pass any component into a cell, it's impractical for us to implement custom focus management as we lack knowledge of the content. In our [docs](https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-analyticaltable--docs#example), this behavior is communicated to users via screen readers using the `cellLabel` column option which can be customized.

You can modify focus handling using a custom [plugin hook](https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-analyticaltable-recipes--docs#how-to-add-your-own-plugin-hooks). I've created a very basic example which you can find [here](https://stackblitz.com/edit/ui5wcr-at-5rtcfo?file=src%2FApp.tsx). However, be aware that making these adjustments may impact a11y, so you need to ensure that screen reader support is maintained. 

I'm going to close this issue now, but feel free to comment any questions or further input here. It won't get lost as, we also monitor closed issues 😉
